test/metabase/upload_test.clj

```diff
@@ -92,30 +104,30 @@
            ["0,0"        0              float-type ",."]
            ["0,0"        0              float-type ", "]
            ["0.0"        0              float-type ".’"]
-           ["$2"         2              int-type]
```

Comment:
why do we have this change?

Reply:
this was because of the bigints problem, but nvm, I've changed it back